webapp/netsoc_admin.py

```diff
@@ -15,17 +15,19 @@
 import sys
 import string
 import register_tools as r
+import help_post as h
+
 
 HOST = "127.0.0.1"
 PORT = "5050"
-DEBUG = False
+DEBUG = True
 
 
 app = flask.Flask(__name__)
 app.secret_key = p.SECRET_KEY
 app.config["SESSION_REFRESH_EACH_REQUEST"] = True
 app.config["SESSION_COOKIE_HTTPONLY"] = True
-app.config["PERMANENT_SESSION_LIFETIME"] = 60 * 10 # seconds
+app.config["PERMANENT_SESSION_LIFETIME"] = 60 * 1000 # seconds
```

Comment:
Why the increase in session time?

Reply:
Because i often get KeyError 'username' which made me think it was getting cleared but this didnt solve the issue, i always had to logout and back in again after already having made a request